libraries/monitor/src/monitor.js

```diff
@@ -120,7 +124,7 @@ class Monitor {
     return (req, res, next) => {
       let sent = false;
       const start = process.hrtime();
-      const send = () => {
+      const send = async () => {
```

Comment:
I see this is now `async` - are we `await`ing something?

Reply:
Yep, the `req.clientId` below need to be awaited now.